installer/webapp/pom.xml

```diff
@@ -155,7 +155,24 @@
             <type>war</type>
         </dependency -->
 
+        <!-- https://mvnrepository.com/artifact/javax.servlet/javax.servlet-api -->
         <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>javax.servlet-api</artifactId>
+            <version>4.0.1</version>
+            <scope>provided</scope>
+        </dependency>
+
+        <!-- https://mvnrepository.com/artifact/javax.servlet.jsp/javax.servlet.jsp-api -->
+        <dependency>
+            <groupId>javax.servlet.jsp</groupId>
+            <artifactId>javax.servlet.jsp-api</artifactId>
+            <version>2.3.3</version>
+            <scope>provided</scope>
+        </dependency>
+
+
+        <!-- <dependency>
```

Comment:
Should this be removed rather than commented out?

Reply:
I do want to remove it at some point.  I left it in there because unlike most groupIds and artifactIds that stay the same and only the version changes these did and I wanted to keep that visual reminder "just in case", at least until testing was 100% complete.

Reply:
Okay, assuming no problems arise I imagine we will want to remove the comments before merging.